assimilation_code/modules/io/io_filenames_mod.f90

```diff
@@ -187,7 +188,7 @@ module io_filenames_mod
 
 end type
 
-character(len=512) :: msgstring
+character(len=512) :: string1, string2, string3
```

Comment:
i'd prefer to leave msgstring alone and add string2 and string 3, to be honest.  trying to look at differences which are just variable renames makes it easy to overlook errors in editing, or masks real code changes.

Reply:
yeah I'm reverting the string changes.  I think there are very few actual changes in this file once you take out the variable name changes.